### DISCUSSION TITLE
How to use a Mobiflight variables ?  increase/decrease don't work if I turn rapidly the knob

### DISCUSSION THREAD

**Discussion by JLD95** [maartentamboer/X-Touch-Mini-FS2020#101] [Q&A]
For A320 (FBW), I would like to use  rotate button in order to increase/decrease lights. I put in my config file  this:

    {
      "index": 7,
      "event_up": "MobiFlight.A32NX_OH_INTEG_LIGHT_INC",
      "event_down": "MobiFlight.A32NX_OH_INTEG_LIGHT_DEC"
    },

but these variables are not found: how to use these mobiflight variables ?

**Comment by JLD95** [comment]
I found this partial solution **which don't work if I turn rapidly the knob**

    {
      "index": 7,
	  "event_up": [ {"event": "MobiFlight.A32NX_OH_INTEG_LIGHT_INC", "type": "manual", "value": 1 },
		            {"event": "MobiFlight.A32NX_ECAM_KNOB_UPPER_INC", "type": "manual", "value": 1 },	  
		            {"event": "MobiFlight.A32NX_ECAM_KNOB_LOWER_INC", "type": "manual", "value": 1 },	 
		            {"event": "MobiFlight.A32NX_PEDEST_KNOB_FLOOD_CPT_INC", "type": "manual", "value": 1 },
		            {"event": "MobiFlight.A32NX_PEDEST_KNOB_FLOOD_FO_INC", "type": "manual", "value": 1 },
		            {"event": "MobiFlight.A32NX_PEDEST_KNOB_INTEG_INC", "type": "manual", "value": 1 },					
		            {"event": "MobiFlight.A32NX_LIGHTS_GLARESHIELD2_INC", "type": "manual", "value": 1 },
		            {"event": "MobiFlight.A32NX_LIGHTS_GLARESHIELD3_INC", "type": "manual", "value": 1 } ], 
	  "event_down": [ {"event": "MobiFlight.A32NX_OH_INTEG_LIGHT_DEC", "type": "manual", "value": 1 },
		              {"event": "MobiFlight.A32NX_ECAM_KNOB_UPPER_DEC", "type": "manual", "value": 1 },	  
		              {"event": "MobiFlight.A32NX_ECAM_KNOB_LOWER_DEC", "type": "manual", "value": 1 },	  
		              {"event": "MobiFlight.A32NX_PEDEST_KNOB_FLOOD_CPT_DEC", "type": "manual", "value": 1 },
		              {"event": "MobiFlight.A32NX_PEDEST_KNOB_FLOOD_FO_DEC", "type": "manual", "value": 1 },
		              {"event": "MobiFlight.A32NX_PEDEST_KNOB_INTEG_DEC", "type": "manual", "value": 1 },					
	  		      {"event": "MobiFlight.A32NX_LIGHTS_GLARESHIELD2_DEC", "type": "manual", "value": 1 },
		              {"event": "MobiFlight.A32NX_LIGHTS_GLARESHIELD3_DEC", "type": "manual", "value": 1 } ]	  
    },


and with additionnal variables for lighting pedestal in **mobiflight-event-module\modules\events.user.txt**

```
// use this file to add your user-defined events
// remove the .inactive extension
// afterwards, the file will not be touched by the next MobiFlight update

//Fly By Wire/A320-Dev/Lighting Pedestal
A32NX_PEDEST_KNOB_FLOOD_CPT_DEC#(A:LIGHT POTENTIOMETER:83,percent) 5 - 0 max 83 (>K:2:LIGHT_POTENTIOMETER_SET)
A32NX_PEDEST_KNOB_FLOOD_CPT_INC#(A:LIGHT POTENTIOMETER:83,percent) 5 + 100 min 83 (>K:2:LIGHT_POTENTIOMETER_SET)
A32NX_PEDEST_KNOB_FLOOD_FO_DEC#(A:LIGHT POTENTIOMETER:76,percent) 5 - 0 max 76 (>K:2:LIGHT_POTENTIOMETER_SET)
A32NX_PEDEST_KNOB_FLOOD_FO_INC#(A:LIGHT POTENTIOMETER:76,percent) 5 + 100 min 76 (>K:2:LIGHT_POTENTIOMETER_SET)
A32NX_PEDEST_KNOB_INTEG_DEC#(A:LIGHT POTENTIOMETER:85,percent) 5 - 0 max 85 (>K:2:LIGHT_POTENTIOMETER_SET)
A32NX_PEDEST_KNOB_INTEG_INC#(A:LIGHT POTENTIOMETER:85,percent) 5 + 100 min 85 (>K:2:LIGHT_POTENTIOMETER_SET)
```


how to have a full solution ?

**Reply by jlr91** [reply]
I think that you have too many events to trigger when you turn rapidly your knob.
But you can ask your question to the Mobiflight discord channel.

**Reply by JLD95** [reply]
Ok thanks I will ask to discord. I think it's like a integer overflow. During rapidly increasing, the buttons switch a bit less half position.
